src/spdx2/agent/spdx2.php

```diff
@@ -527,10 +527,14 @@ protected function addCopyrightResults(&$filesWithLicenses, $uploadId)
     /* @var $copyrightDao CopyrightDao */
     $copyrightDao = $this->container->get('dao.copyright');
     $uploadtreeTable = $this->uploadDao->getUploadtreeTableName($uploadId);
-    $allEntries = $copyrightDao->getAllEntries('copyright', $uploadId, $uploadtreeTable, $type='statement');
-    foreach ($allEntries as $finding) {
+    $allScannerEntries = $copyrightDao->getScannerEntries('copyright', $uploadtreeTable, $uploadId, $type='statement', $extrawhere=null);
```

Comment:
You can use the existing function getAllEntriesReport from copyrightDao to get both scanner and edited entries.

Reply:
Hello,

thanks for your comment.
I tried to use getAllEntriesReport, but this function merges scannerEntries and editedEntries arrays in a way it's hard to use them in the report (For example: we have 1 scanner and 5 user-added copyrights - result of this function is array having 5 rows with user-edited copyrights but for each row there is also 5 times duplicated scanner copyright)
I hope this explanation is clear enought - if not, I can show you an example :)

That's why I decided to use 2 functions separately (funcion getAllEntriesReport function is used also by Unified Report and i decided no to modify it)

Reply:
ok